client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -222,8 +223,9 @@ const MailboxSelectionList = ( { domains } ) => {
 	} );
 
 	useEffect( () => {
+		const recorder = withEnhancers( recordPageView, [ enhanceWithSiteMainProduct ] );
```

Comment:
The only downside here is that the new event properties might be missing from some `calypso_page_view` events, so we'll have to make sure we trigger every `recordPageView()` of interest specifically with this enhancer.

Reply:
This is now applied universally! https://github.com/Automattic/wp-calypso/pull/66731/files#r954435072